frontend/src/js/components/settings/planselection.js

```diff
@@ -11,82 +11,119 @@
         {Object.values(PLANS).map(item => (
           <div
             key={item.id}
             className={`planPanel ${classes.planPanel} ${updatedPlan === item.id ? 'active' : ''} ${isTrial || canUpgrade(item.id) ? '' : 'muted'}`}
-            onClick={() => onPlanSelect(item.id)}
           >
-            {!isTrial && canUpgrade(item.id) && (
-              <div className={`uppercased align-center muted ${classes.planNote}`}>{item.id === currentPlan ? 'current plan' : 'upgrade'}</div>
-            )}
-            <h4>
-              {item.name} {item.offer && isTrial && offerValid ? offerTag : null}
-            </h4>
             <div>
-              {item.offer && isTrial && offerValid ? (
-                <>
-                  <div className={`link-color bold ${classes.price}`}>{item.offerprice}</div>
-                  <div className="pre-line">{item.price2}</div>
-                </>
-              ) : (
-                <>
-                  <div className={`link-color bold ${classes.price}`}>{item.price}</div>
-                  <div>{item.deviceCount}</div>
-                </>
-              )}
+              <p className={`${classes.planName}`}>Mender {item.name}</p>
+              <h4 className={`${classes.planPrice}`}>{item.price}</h4>
+              <div className={`margin-bottom`}>
+                <div className={`flexbox align-center`}>
+                  <DeveloperBoardIcon fontSize="medium" className={`${classes.planIcon}`} />
+                  <div className={`${classes.devicesCount}`}>
+                    {item.deviceCount}
+                    {item.id === 'enterprise' ? '' : '*'}
+                  </div>
+                </div>
+              </div>
+              <ul className="unstyled">
+                {item.features.map((feature, index) => (
+                  <li className={`flexbox ${classes.planFeature}`} key={`${item.id}-feature-${index}`}>
+                    <CheckIcon className={classes.planIcon} />
+                    <div className={classes.planFeatureText}>{feature}</div>{' '}
+                  </li>
+                ))}
+              </ul>
             </div>
-            <ul className="unstyled">
-              {item.features.map((feature, index) => (
-                <li key={`${item.id}-feature-${index}`}>
-                  <InfoText variant="dense">{feature}</InfoText>
-                </li>
-              ))}
-            </ul>
+            <Button
+              onClick={() => onPlanSelect(item.id)}
+              variant="contained"
+              color={item.id === 'enterprise' ? 'secondary' : 'primary'}
+              disabled={
+                (item.id === currentPlan && !isTrial) ||
+                (isUpgradePending && item.id === isUpgradePending.plan) ||
+                (currentPlan === 'enterprise' && item.id !== 'enterprise') ||
+                (currentPlan === 'professional' && item.id === 'os')
+              }
+              className={`${classes.plantton}`}
```

Comment:
```suggestion
              className={classes.plantton}
```
and the class referenced is probably not right